hyperon_das/client.py

```diff
@@ -32,110 +28,50 @@ def _send_request(self, payload) -> str | dict | int:
         except requests.exceptions.RequestException as e:
             raise e
 
-    def get_node(
-        self,
-        node_type: str,
-        node_name: str,
-        output_format: QueryOutputFormat = QueryOutputFormat.HANDLE,
-    ) -> Union[str, Dict]:
+    def get_node_handle(self, node_type: str, node_name: str) -> str:
```

Comment:
This is wrong. We don't need to delegate `get_node_handle()` to the remote DAS. This function shouldn't be here. Node and links handles can be calculated using a static method in the basic AtomDB. Se my design for reference.